src/less/tooltip/ds6/tooltip.less

```diff
@@ -4,151 +4,153 @@
+            &__arrow-top-right {
+                .tooltip-arrow-top;
 
-            right: 12px;
-        }
+                right: 12px;
+            }
 
-        &__pointer-bottom-left,
-        &__arrow-bottom-left {
-            .tooltip-arrow-bottom;
+            &__arrow-bottom-left {
+                .tooltip-arrow-bottom;
 
-            left: 16px;
-        }
+                left: 16px;
+            }
 
-        &__pointer-bottom-center,
-        &__arrow-bottom-center {
-            .tooltip-arrow-bottom;
+            &__arrow-bottom-center {
+                .tooltip-arrow-bottom;
 
-            left: calc(50% - 8px);
-        }
+                left: calc(50% - 8px);
+            }
 
-        &__pointer-bottom-right,
-        &__arrow-bottom-right {
-            .tooltip-arrow-bottom;
+            &__arrow-bottom-right {
+                .tooltip-arrow-bottom;
 
-            right: 16px;
-        }
+                right: 16px;
+            }
 
-        &__arrow-left-top {
-            .tooltip-arrow-left;
+            &__arrow-left-top {
+                .tooltip-arrow-left;
 
-            top: 12px;
-        }
+                top: 12px;
+            }
 
-        &__arrow-left-center {
-            .tooltip-arrow-left;
+            &__arrow-left-center {
+                .tooltip-arrow-left;
 
-            top: calc(50% - 8px);
-        }
+                top: calc(50% - 8px);
+            }
 
-        &__arrow-left-bottom {
-            .tooltip-arrow-left;
+            &__arrow-left-bottom {
+                .tooltip-arrow-left;
 
-            bottom: 12px;
-        }
+                bottom: 12px;
+            }
 
-        &__arrow-right-top {
-            .tooltip-arrow-right;
+            &__arrow-right-top {
+                .tooltip-arrow-right;
 
-            top: 12px;
-        }
+                top: 12px;
+            }
 
-        &__arrow-right-center {
-            .tooltip-arrow-right;
+            &__arrow-right-center {
+                .tooltip-arrow-right;
 
-            top: calc(50% - 8px);
-        }
+                top: calc(50% - 8px);
+            }
 
-        &__arrow-right-bottom {
-            .tooltip-arrow-right;
+            &__arrow-right-bottom {
+                .tooltip-arrow-right;
 
-            bottom: 12px;
+                bottom: 12px;
+            }
         }
     }
 }
 
-.tooltip {
-    &__overlay {
-        position: relative;
+.tooltip__content {
+    align-items: center;
+    display: inline-flex;
+    max-width: 344px;
+
+    &-heading {
+        margin: 0 0 16px 0;
     }
 
-    &__content {
-        align-items: center;
-        display: inline-flex;
-        max-width: 344px;
+    p {
+        margin: 0;
 
-        &-heading {
-            margin: 0 0 16px 0;
-        }
-
-        p {
-            margin: 0;
-
-            &:not(:first-of-type) {
-                margin-top: 4px;
-            }
+        &:not(:first-of-type) {
+            margin-top: 4px;
```

Comment:
I actually think this could be `0.25em` since it's based on the line-height of the paragraphs. Thoughts?